ix-dev/community/urbackup/templates/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.urbackup_container_name, "image") %}
+
+{% do c1.set_user(values.run_as.user, values.run_as.group) %}
+{% do c1.add_caps(["CHOWN", "SETUID", "SETGID"]) %}
+{% do c1.healthcheck.set_test("http", {"port": values.network.web_port}) %}
+
+
+{% if values.urbackup.zfs_enabled %}
+  {% do c1.devices.add_device("/dev/zfs", "/dev/zfs") %}
+{% endif %}
+
+{% if values.urbackup.btrfs_enabled %}
+  {% do c1.add_caps(["SYS_ADMIN"]) %}
+{% endif %}
```

Comment:
Truenas system does not even have btrfs. So not sure about this either.

Reply:
ill remove

Reply:
fixed